test/sharness/t0170-routing-dht.sh

```diff
@@ -1,128 +0,0 @@
-    HASH="$(echo "hello world" | ipfsi 2 add -q)" &&
-    ipfsi 2 name publish "/ipfs/$HASH" &&
-    ipfsi 1 routing get "/ipns/$PEERID_2" >get_result
-  '
-
-  test_expect_success 'get with good keys contains the right value' '
-    cat get_result | grep -aq "/ipfs/$HASH"
-  '
-
-  test_expect_success 'put round trips (#3124)' '
-    ipfsi 0 routing put "/ipns/$PEERID_2" get_result | sort >putted &&
-    [ -s putted ] ||
-    test_fsh cat putted
-  '
-
-  test_expect_success 'put with bad keys fails (issue #5113)' '
-    ipfsi 0 routing put "foo" <<<bar >putted
-    ipfsi 0 routing put "/pk/foo" <<<bar >>putted
-    ipfsi 0 routing put "/ipns/foo" <<<bar >>putted
-    [ ! -s putted ] ||
-    test_fsh cat putted
-  '
-
-  test_expect_success 'put with bad keys returns error (issue #4611)' '
-    test_must_fail ipfsi 0 routing put "foo" <<<bar &&
-    test_must_fail ipfsi 0 routing put "/pk/foo" <<<bar &&
-    test_must_fail ipfsi 0 routing put "/ipns/foo" <<<bar
-  '
-
-  test_expect_success 'get with bad keys (issue #4611)' '
-    test_must_fail ipfsi 0 routing get "foo" &&
-    test_must_fail ipfsi 0 routing get "/pk/foo"
-  '
-
-  test_expect_success "add a ref so we can find providers for it" '
-    echo "some stuff" > afile &&
-    HASH=$(ipfsi 3 add -q afile)
-  '
-
-  # ipfs routing findprovs <key>
-  test_expect_success 'findprovs' '
-    ipfsi 4 routing findprovs $HASH > provs &&
-    iptb attr get 3 id > expected &&
-    test_cmp provs expected
-  '
-
-  # ipfs routing get --enc=json has correct properties
-  test_expect_success 'routing get --enc=json has correct properties' '
-    HASH="$(echo "hello world" | ipfsi 2 add -q)" &&
-    ipfsi 2 name publish "/ipfs/$HASH" &&
-    ipfsi 1 routing get --enc=json "/ipns/$PEERID_2" | jq -e "has(\"Extra\") and has(\"Type\")"
-  '
-
-  # ipfs dht query <peerID>
-  #
-  # We test all nodes. 4 nodes should see the same peer ID, one node (the
-  # closest) should see a different one.
-
-  for i in $(test_seq 0 4); do
-    test_expect_success "dht query from $i" '
-      ipfsi "$i" dht query "$HASH" | head -1 >closest-$i
-    '
-  done
-
-  test_expect_success "collecting results" '
-    cat closest-* | sort | uniq -c | sed -e "s/ *\([0-9]\+\) .*/\1/g" | sort -g > actual &&
-    echo 1 > expected &&
-    echo 4 >> expected
-  '
-
-  test_expect_success "checking results" '
-    test_cmp actual expected
-  '
-
-  test_expect_success 'stop iptb' '
-    iptb stop
-  '
-
-  test_expect_success "dht commands fail when offline" '
-    test_must_fail ipfsi 0 routing findprovs "$HASH" 2>err_findprovs &&
-    test_must_fail ipfsi 0 routing findpeer "$HASH" 2>err_findpeer &&
-    test_must_fail ipfsi 0 routing put "/ipns/$PEERID_2" "get_result" 2>err_put &&
-    test_should_contain "this command must be run in online mode" err_findprovs &&
-    test_should_contain "this command must be run in online mode" err_findpeer &&
-    test_should_contain "this action must be run in online mode" err_put
-  '
-}
```

Comment:
Did thoses tests were not ported ? `dht commands fail when offline` I can see why but `ipfs dht query <peerID>` could have some value. You could argue this isn't needed to be tested here since the dht unit tests should cover this.

Reply:
The query tests were ported to dht_legacy_test.go in https://github.com/ipfs/kubo/commit/bfa425fc67b5a6125412cd2453d6fa7f83a4bc96#diff-41cd711687aeefc5dc1c13921ca53bb62b254b86d433c08a3b04ea1beb641709 , they don't really belong here as they're not for `ipfs routing`.